pkg/app/piped/controller/planner.go

```diff
@@ -325,3 +342,17 @@ func (p *planner) reportDeploymentCancelled(ctx context.Context, commander, reas
 	}
 	return err
 }
+
+func (p *planner) getMentionedAccounts(ctx context.Context, event model.NotificationEventType, targetDSP deploysource.Provider) ([]string, error) {
+	ds, err := targetDSP.GetReadOnly(ctx, ioutil.Discard)
+	if err != nil {
+		return nil, fmt.Errorf("failed to prepare running deploy source data (%w)", err)
+	}
+
+	for _, v := range ds.GenericDeploymentConfig.DeploymentNotification.Mentions {
```

Comment:
This can cause panic. We'd better ensure `DeploymentNotification` is non-nil.

Reply:
2c0be74